### ISSUE TITLE
I thibk that for accessibility reasons you should have an alt tag, even if it is empty

### ISSUE THREAD

**Issue by kira-abl** [facn6/onePageChallenge.io#34]
https://github.com/facn6/onePageChallenge.io/blob/bc849a72d85cadbb065b67d8d7feaf8d2c5cfb8c/index.html#L38
